templates/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Car Plate Recognition</title>
+		<link rel="stylesheet" type="text/css" href="../static/css/style.css">
+	</head>
+	<body>
+		<div class="container">
+      <h1>Car Plate Recognition</h1>
+      <div class="form-group">
+        <label for="upload">Select an car image to upload:</label>
```

Comment:
or u  can say: "upload a car image"

Reply:
I use 'select' term to avoid the misleading